packages/noco-docs/content/en/developer-resources/rest-apis.md

```diff
@@ -194,6 +194,7 @@ Currently, the default value for {orgs} is <b>noco</b>. Users will be able to ch
 |---|---|---|
 | eq | equal | (colName,eq,colValue) |
 | not | not equal | (colName,not,colValue) |
+| neq | not equal | (colName,neq,colValue) |
```

Comment:
You can add `not` as an alias of `neq` in meaning column

Reply:
Done